### PR TITLE
Clarify the criteria to assign reviewers in the PR template

### PR DESCRIPTION
**Pull request by g-votte**
This PR clarifies in `PULL_REQUEST_TEMPLATE.md` some expectations for a pull request to be assigned reviewers.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- Thank you for creating a pull request! In general, we merge your pull requests after they get two or more approvals. -->
+<!-- Thank you for creating a pull request! In general, we merge your pull request after it gets two or more approvals. To proceed to the review process by the maintainers, please make sure that the PR meets the following conditions: (1) it passes all CI checks, and (2) it is neither in the draft nor WIP state. If you wish to discuss the PR in the draft state or need any other help, please mention the Optuna development team in the PR. -->
```
Comment (toshihikoyanase) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: [Question] I guess we don't have the development team in the Optuna org. How do contributors find the names of the maintainers?

> please mention the Optuna development team in the PR

Comment (g-votte) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Right; so far, the contributors have to guess watching the reviewers of other PRs. As a followup task, we've better create a team or provide an official list of maintainers somewhere in the repository.

**Comment by toshihikoyanase** [comment]
GitHub Actions is back now. Could you rerun the CI jobs, please?

**Comment by toshihikoyanase** [comment]
@contramundum53 Could you review this PR, please? The change does not affect the code base, and I think we can merge it with a single approval. Please feel free to remove the assignment if you are busy.

**Review by contramundum53 — APPROVED** [review]
LGTM!

**Pull request merged**